lib/ngx/process.lua

```diff
@@ -0,0 +1,35 @@
+-- Copyright (C) Yichun Zhang (agentzh)
+
+
+local ffi = require 'ffi'
+local base = require "resty.core.base"
+local getfenv = getfenv
+
+
+local C = ffi.C
+local _M = { version = base.version }
+
+
+ffi.cdef[[
+void ngx_http_lua_ffi_worker_privileged(int enable);
```

Comment:
@membphis I don't think there is any use in disabling this privileged agent since it is disabled by default anyway. So we can omit the parameter as well.